Make scan work with -vvv

noahl opened the pull request:
Temporary workaround to make scan command work with increased Ansible
verbosity.

Commit:
Make scan work with -vvv

Temporary workaround to make scan command work with increased Ansible
verbosity.

chambridge commented:
Lint error:
```
************* Module rho.scancommand

W: 44,53: Unused argument 'ansible_verbosity' (unused-argument)
```

Commit:
Add pylint disable for -vvv workaround

Have to disable unused-argument checking for _create_ping_inventory
because the workaround is to ignore an argument.

noahl commented:
@chambridge should be fixed now
